### PR TITLE
Feature/private entries

### PR DESCRIPTION
**Pull request by agduncan94**
* Re-adds support for private tool icon to search and tool page
* Small UI enhancement to verified checkmarks
* Cleaned up my tools/my workflows accordion

### CODE REVIEW

File: src/ds-style-fix.scss [hunk 1]
```diff
@@ -663,3 +663,16 @@ div.contact-button {
   margin-left: auto;
   margin-right: auto;
 }
+
+.private-lock {
+  color: #487980;
+}
+
+.accordion-hr {
```
Comment (garyluu) [inline src/ds-style-fix.scss]: Rename to .my-1 or .my-2, etc for when we ever switch to https://v4-alpha.getbootstrap.com/utilities/spacing/

File: src/app/container/container.component.html [hunk 2]
```diff
@@ -27,13 +27,16 @@
   </div>
 </div>
 <div class="row">
-  <div class="col-md-7">
+  <div class="col-md-12">
```
Comment (garyluu) [inline src/app/container/container.component.html]: This pushes the publish and refresh buttons down in another row below the title.   Not sure if that's intended

File: src/app/shared/refresh-organization/refresh-organization.component.html [hunk 3]
```diff
@@ -14,5 +14,5 @@
   ~    limitations under the License.
   -->
 
-<button tooltip="Refresh organization" placement="right" type="button" class="btn btn-primary btn-xs glyphicon glyphicon-refresh" (click)="refreshOrganization()" [disabled]="toDisable()">
+<button tooltip="Refresh organization" placement="right" style="margin-right:5px;" type="button" class="btn btn-primary btn-xs glyphicon glyphicon-refresh" (click)="refreshOrganization()" [disabled]="toDisable()">
```
Comment (garyluu) [inline src/app/shared/refresh-organization/refresh-organization.component.html]: We've generally avoided inline styles.  Rename to .mr-1 or .my-2, etc for when we ever switch to https://v4-alpha.getbootstrap.com/utilities/spacing/

File: src/app/workflow/workflow.component.html [hunk 4]
```diff
@@ -27,10 +27,10 @@
   </div>
 </div>
 <div class="row">
-  <div class="col-md-7">
+  <div class="col-md-12">
```
Comment (garyluu) [inline src/app/workflow/workflow.component.html]: Same as container

File: src/app/container/container.component.html (hunk 2 above)
Comment (agduncan94) [reply src/app/container/container.component.html]: Oops

**Pull request merged**